authentication/managing_cloud_provider_credentials/cco-short-term-creds.adoc

```diff
@@ -53,11 +53,9 @@ include::modules/cco-short-term-creds-component-permissions-aws.adoc[leveloffset
 //OLM-managed Operator support for authentication with AWS STS
 include::modules/cco-short-term-creds-aws-olm.adoc[leveloffset=+2]
 
-////
 [role="_additional-resources"]
 .Additional resources
-* xr\ef:../../operators/operator_sdk/osdk-token-auth.html#osdk-aws-sts_osdk-token-auth[CCO-based workflow for OLM-managed Operators with AWS STS]
-////
+* xref:../../operators/operator_sdk/osdk-token-auth.adoc#osdk-cco-aws-sts_osdk-token-auth[CCO-based workflow for OLM-managed Operators with AWS STS]
```

Comment:
@jeana-redhat FYI, updating link for prime-time.

Reply:
thank you so much for relieving me of remembering to do a thing :bow: